cap/modules/theme/templates/cap_theme/header_login.html

```diff
@@ -25,9 +25,9 @@
 
 {% block navbar_login %}
   {%- if not current_user.is_authenticated %}
-  <form class="navbar-right">
-    <a  href=" {{url_for('invenio_oauthclient.login', remote_app='cern', next=url_for('cap_theme.index'))}}" class="btn btn-default sign-in-btn btn-lg" >{{_('Sign in with CERN')}}</a>
-  </form>
+  <div>
```

Comment:
Here you can add a class and then just use .name-of-class .login-button { override here what needs to be overriden from .login-button class }. That way we can only have a login-button class and we can get rid completely the id.